stable/mlrun-kit/requirements.yaml

```diff
@@ -3,7 +3,7 @@ dependencies:
   version: "0.7.19"
   repository: "https://nuclio.github.io/nuclio/charts"
 - name: mlrun
-  version: "0.5.16"
+  version: "0.6.1"
```

Comment:
why not 0.6.3 (latest) ?

Reply:
It's the chart version - the latest for it is 0.6.3